packages/portal-style/src/components/_forms.scss

```diff
@@ -14,6 +14,7 @@
   .form-text {
     font-size: 12px;
     color: $grey-dark;
+    white-space: pre-line;
```

Comment:
Do we need `pre-line` there ? A regular `normal` should be enough

Reply:
It didn't work for line break